<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SL-ENG77-52LJL" ignored="0" name="Client Admin - ServiceSubCategory" tests="1" failures="1" timestamp="30 Apr 2019 08:02:41 GMT" time="5.408" errors="0">
  <testcase name="ServiceSubCategoryCreation" time="0.001" classname="office.sirion.suite.clientAdmin.ServiceSubCategory">
    <failure type="org.testng.internal.reflect.MethodMatcherException" message="
Data provider mismatch
Method: ServiceSubCategoryCreation([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=2, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=3, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) Account Payable,(java.lang.String) Fixed Fee,(java.lang.String) Fixed Fee]">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
Data provider mismatch
Method: ServiceSubCategoryCreation([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=2, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=3, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) Account Payable,(java.lang.String) Fixed Fee,(java.lang.String) Fixed Fee]
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- ServiceSubCategoryCreation -->
</testsuite> <!-- Client Admin - ServiceSubCategory -->
